pkg/util/util.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2013 The Prometheus Authors
```

Comment:
let's not start a "util" kitchen-sink package – all this does is parsing addresses so I would call it "address" or something. Content-wise this package is fine, but names have power and if we call it util it will accumulate all kinds of stuff because we stop forcing ourselves to think about proper package scopes in the future.